src/components/NavigationDrawer.vue

```diff
@@ -44,6 +46,14 @@ import {
 
 export default {
   name: 'NavigationDrawer',
+  setup() {
+    // Vuetify has its own translation layer that isn't very good
+    // It seems to conflict with the namespace of vue-i18n so we need to import it at the component
+    // level and utilize it this way with a composable. There may be another more global way but
+    // I haven't found anything just yet
+    const { t } = useI18n();
+    return { $t: t };
+  },
```

Comment:
This isn't the worst, but does give me a little big of pause. I think there's a way to get the `$t` global in all components. While I don't want to load EVERY translation up front, I think some of them should be global (shared text from buttons that are reused, etc).